jwt/src/main/java/com/springsecuritypractice/jwt/config/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,19 @@
+package com.springsecuritypractice.jwt.config;
```

Comment:
I would rename the package
```suggestion
package com.springsecuritypractice.jwt.security;
```